packages/tools/src/utilities/math/polyline/simplify.ts

```diff
@@ -0,0 +1,105 @@
+import type { Types } from '@cornerstonejs/core';
+import * as mathLine from '../line';
+
+const DEFAULT_EPSILON = 0.1;
+
+/**
+ * Ramer–Douglas–Peucker algorithm implementation to decimate a polyline
+ * to a similar polyline with fewer points
+ *
+ * https://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm
+ * https://rosettacode.org/wiki/Ramer-Douglas-Peucker_line_simplification
+ * https://karthaus.nl/rdp/
+ *
+ * @param polyline - Polyline to simplify
+ * @param epsilon - A maximum given distance 'epsilon' to decide if a point
+ * should or shouldn't be added the the simplified polyline version. In each
+ * iteration the polyline is split into two polylines and the distance of each
+ * point from those new polylines are checked against the line that connects
+ * the first and last points.
+ * @returns Simplified polyline
```

Comment:
is this our implementatoin? is the used code open permissive license?

Reply:
This is 100% my implementation.